library/psa_crypto_ecp.c

```diff
@@ -101,8 +101,17 @@ psa_status_t mbedtls_psa_ecp_load_representation(
     mbedtls_ecp_keypair_init(ecp);
 
     /* Load the group. */
-    grp_id = mbedtls_ecc_group_of_psa(PSA_KEY_TYPE_ECC_GET_FAMILY(type),
-                                      curve_bits, !explicit_bits);
+    if (PSA_KEY_TYPE_ECC_GET_FAMILY(type) == PSA_ECC_FAMILY_MONTGOMERY)
+    {
```

Comment:
Im not sure why this is needed here. It looks like that function will return MBEDTLS_ECP_DP_CURVE25519 for both 255 and 256 bits.

Reply:
Yes, current value of "bits" in case CURVE25519 is 256. Without this modification mbedtls_ecc_group_of_psa can not return MBEDTLS_ECP_DP_CURVE25519. 
I see it also have no affect to other curve type.